addons/project/models/project_task_type.py

```diff
@@ -27,8 +27,7 @@ def _get_default_project_ids(self):
         string='Email Template',
         domain=[('model', '=', 'project.task')],
         help="If set, an email will be automatically sent to the customer when the task reaches this stage.")
-    fold = fields.Boolean(string='Folded in Kanban',
-        help='If enabled, this stage will be displayed as folded in the Kanban view of your tasks. Tasks in a folded stage are considered as closed.')
+    fold = fields.Boolean(string='Folded in Kanban')
```

Comment:
Can you also reword your commit message?

```txt
[IMP] project: task stage improvement in the project app

This commit removes the tooltip of the `fold` field because
the second part is no longer true and the first part does not
provide any added value.
It also makes `project_ids` field required in the form view of
Task Stage model (`project.task.type`) to force the user to link
his new stage to a project. Otherwise, the stage won't ever be
visible anywhere in the interface. Indeed, stages not holding 
tasks are hidden in cross-projects views.

task-3251621
```